src/main/java/com/izmaylov/module2/model/Product.java

```diff
@@ -0,0 +1,73 @@
+package com.izmaylov.module2.model;
+
+import lombok.Getter;
+import lombok.ToString;
+
+@Getter
+@ToString
+public class Product{
+    private String type;
+    private String series;
+    private String screenType;
+    private int price;
+    private String model;
+    private String diagonal;
+    private String country;
+    private boolean emptyValue;
```

Comment:
Nice solution, but the fields model, diagonal, country are implementation-specific